src/main/java/frc/robot/RobotContainer.java

```diff
@@ -12,39 +12,47 @@
 import frc.robot.subsystems.elevator.ElevatorIO;
 import frc.robot.subsystems.elevator.ElevatorIOReal;
 import frc.robot.subsystems.elevator.ElevatorIOSim;
+import frc.robot.subsystems.gripper.*;
 import frc.robot.swerve.SwerveDrive;
 
 public class RobotContainer {
 
     private static RobotContainer INSTANCE = null;
     private final Elevator elevator;
+    private final Gripper gripper;
     private final Conveyor conveyor;
     private final SwerveDrive swerveDrive;
     private final CommandXboxController xboxController = new CommandXboxController(0);
     private final CommandGenericHID keyBoardControl = new CommandGenericHID(0);
 
     /** The container for the robot. Contains subsystems, OI devices, and commands. */
     private RobotContainer() {
+        GripperIO gripperIO;
         ElevatorIO elevatorIO;
         ConveyorIO conveyorIO;
         ConveyorConstants.initialize(Constants.CURRENT_MODE);
         switch (Constants.CURRENT_MODE) {
             case REAL:
                 elevatorIO = new ElevatorIOReal();
                 conveyorIO = new ConveyorIOSim();
+                gripperIO = new GripperIOReal();
                 break;
             case SIM:
             case REPLAY:
             default:
                 conveyorIO = new ConveyorIOSim();
                 elevatorIO = new ElevatorIOSim();
+                gripperIO = new GripperIOSim();
                 break;
         }
         Elevator.initialize(elevatorIO);
         Conveyor.initialize(conveyorIO);
+        Gripper.initialize(gripperIO);
         Constants.initSwerve();
         Constants.initVision();
+        GripperConstants.initConstants();
```

Comment:
This needs to happen before creating the io, because the io uses the values initialized here.

Reply:
was fixed long ago